crates/icrate/examples/browser.rs

```diff
@@ -0,0 +1,284 @@
+type BoxError = Box<dyn std::error::Error + Send + Sync + 'static>;
```

Comment:
Nit: Add `#![deny(unsafe_op_in_unsafe_fn)]`. I get that it's tempting to do `unsafe fn run()`, but I'd much rather actually mark the methods we're using here as safe inside `header-translator/src/data`